cmd/flow-aggregator/flow-aggregator.go

```diff
@@ -28,11 +32,25 @@ import (
 
 	"antrea.io/antrea/pkg/clusteridentity"
 	aggregator "antrea.io/antrea/pkg/flowaggregator"
+	"antrea.io/antrea/pkg/flowaggregator/kafka"
 	"antrea.io/antrea/pkg/log"
 	"antrea.io/antrea/pkg/signals"
 )
 
-const informerDefaultResync = 12 * time.Hour
+var (
+	// certDir is the directory that the TLS Secret should be mounted to. Declaring it as a variable for testing.
+	certDir = "/var/run/antrea/flow-aggregator-kafka-tls"
+	// certReadyTimeout is the timeout we will wait for the TLS Secret being ready. Declaring it as a variable for testing.
+	certReadyTimeout = 2 * time.Minute
+)
+
+const (
+	// The names of the files that should contain the CA certificate and the TLS key pair.
+	CACertFile            = "ca.crt"
+	TLSCertFile           = "tls.crt"
+	TLSKeyFile            = "tls.key"
```

Comment:
just to clarify, ca.crt is to authenticate the Kafka broker, while tls.crt and tls.key are to authenticate the aggregator to the broker?

maybe this should be covered by comments

Reply:
There are two different options. Added comments.